services/localfile/client/cli-controllers/file.utils.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2024 Snowflake Inc. All rights reserved.
+
+	Licensed under the Apache License, Version 2.0 (the
+	"License"); you may not use this file except in compliance
+	with the License.  You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing,
+	software distributed under the License is distributed on an
+	"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+	KIND, either express or implied.  See the License for the
+	specific language governing permissions and limitations
+	under the License.
+*/
+
+package cli_controllers
```

Comment:
I'd recommend keeping these in client.go to keep it consistent with the other localfile commands.

Reply:
It cause too much code in client, for me it hard to follow and hard to find definition of exact command what I need. By keep it in separate file I:
- improve readability
- simplify search of the particular cli handler definition

Have 1 component per file it common best practice for a lot of languages. 

What the advantage of keeping code in client.go approach?

Reply:
I'd say keeping it in client.go is much more simpler approach, and wouldn't introduce new variances. I don't see anything wrong with long, but well-structured, file. On the other hand, breaking it into many files would make it harder to navigate.
Also if we decide to refactor, let's do it for all other commands.

Reply:
It subjective. For me it harder to navigate in big file. When I see clear logical structure of the whole project in much easier for me. Mostly it matter of habit, I suggest to try reference only solid facts and measurable pros and cons for a project.

solid fact from my side:
- devs would work with single command at a time. Other commands code not necessary for them
- to find exact command devs spend less time if it in separate files, lets have an example, lets assume we need to find command controller, but we not remember its exact name:
   - right now:
      - open relevant service
      - open client.go
      - scroll thought a hundreds lines of code to find command controller definition
    - with 1 command per file
      - open relevant service
      - open folder with commands handlers
      - go thought <20 file names to find command controller definition